specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json

```diff
@@ -0,0 +1,451 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-01-01",
+    "title": "Marketplace RP Service"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/providers/Microsoft.Marketplace/PrivateStores/{PrivateStoreId}/Offers": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/PrivateStoreIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "get": {
```

Comment:
>get [](start = 7, length = 3)

all operations should have "operationId"  (I see only one of the operations has this defined)

Get individual resource type :
     "operationId": "TypeName_Get",

Collection GET:
    "operationId": "TypeName_List",

PUT:
"operationId": "TypeName_CreateOrUpdate",

PATCH:
"operationId": "TypeName_Update",


delete:
"operationId": "TypeName_Delete",


Please referto other swagger specs in the repo for reference, if required.

Reply:
done